### PR TITLE
added J1 case to multiple_of_order() in torsion_subgroup.py

### PR DESCRIPTION
**Pull request by kevinywlui**
New test cases are tested against Magma here: https://cloud.sagemath.com/projects/0d310a14-a099-47d9-87c7-df6b63af54a6/files/worksheets/2016-05-23-140710-magma.sagews

### CODE REVIEW

File: sage_modabvar/torsion_subgroup.py [hunk 1]
```diff
@@ -332,6 +335,11 @@ def multiple_of_order(self, maxp=None):
             sage: G = J.rational_torsion_subgroup()
             sage: G.multiple_of_order(11)
             5
+
+        Increasing maxp will yield a tighter bound. If maxp=None, then Sage
```
Comment (williamstein) [inline sage_modabvar/torsion_subgroup.py]: will --> may

**Comment by williamstein** [comment]
Wow nice job figuring out the charpoly stuff and relative numbers fields.  How confident of it are you?  It looks pretty tricky!

**Comment by kevinywlui** [comment]
Pretty confident. We can also compute frobenius using the Gp polynomials. I did that here and checked it against Magma and Sage https://cloud.sagemath.com/projects/0d310a14-a099-47d9-87c7-df6b63af54a6/files/worksheets/2016-06-18-110312-prs.sagews

You used the 3 successive prime method as default in Sage but in Magma, you used maxp=50 as default. Do you happen to remember why you made these choices?

**Comment by kevinywlui** [comment]
Oh! I just realized that I can compute frobenius without computing the Galois closure by computing it at a few points using the Gps then interpolating.

**Comment by williamstein** [comment]
They are completely arbitrary choices -- you could do some actual computations and make a table (say up to some level) and come up with a better choice.  Simon Spicer did that sort of thing with the parameters for the algorithms of his thesis and it was quite interesting.

**Comment by kevinywlui** [comment]
Sounds fun. I'll play around with it.

**Pull request merged**